README.md

```diff
@@ -6,21 +6,21 @@ This repo provides a package you can use to build with XMTP in a React Native or
 
 Is there a feature you need that's currently supported? Please [open an issue](https://github.com/xmtp/xmtp-react-native/issues).
 
-Or better yet, open a PR and we'll get it reviewed and merged as soon as possible. If you contribute a PR that gets merged into this repo, you'll be eligible to [claim this XMTP contributor POAP](https://www.gitpoap.io/gp/1042)!
+Or better yet, open a PR and we'll get it reviewed and merged as soon as possible. If you contribute a PR that gets merged into this repo, you'll be eligible to [claim this XMTP contributor POAP](https://www.gitpoap.io/gp/1100).
 
-To learn more about XMTP and get answers to frequently asked questions, see [XMTP documentation](https://xmtp.org/docs).
+## Documentation
 
-![x-red-sm](https://user-images.githubusercontent.com/510695/163488403-1fb37e86-c673-4b48-954e-8460ae4d4b05.png)
+To learn how to use the XMTP React Native SDK and get answers to frequently asked questions, see the [XMTP documentation](https://docs-xmtp-org-git-v3-preview-ephemerahq.vercel.app/).
 
-## Reference docs
+## SDK reference
 
-Access the `xmtp-react-native` client SDK [reference documentation](https://xmtp.github.io/xmtp-react-native/modules.html).
+Access the [XMTP React Native SDK reference documentation](https://xmtp.github.io/xmtp-react-native/modules.html).
```

Comment:
Can you spot-check these to see if these are building as expected, please? TY!

Reply:
Yep look good!